tests/sentry/incidents/action_handlers/test_opsgenie.py

```diff
@@ -94,11 +94,13 @@ def test_build_incident_attachment(self):
         metric_value = 1000
         data = build_incident_attachment(
             alert_context=AlertContext.from_alert_rule_incident(incident.alert_rule),
-            open_period_identifier=incident.identifier,
+            metric_issue_context=MetricIssueContext.from_legacy_models(
+                incident=incident,
+                new_status=IncidentStatus(incident.status),
+                metric_value=metric_value,
+            ),
             organization=incident.organization,
-            snuba_query=incident.alert_rule.snuba_query,
-            new_status=IncidentStatus(incident.status),
-            metric_value=metric_value,
+            notification_uuid="",
```

Comment:
nit: this is an optional arg, if you're just passing an empty string feels like you may as well not pass it at all